.github/workflows/ci.yaml

```
@@ -45,7 +45,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu]
-        ruby: ['2.7', '3.0', '3.1', '3.2', '3.3', 'ruby-head', 'debug', 'truffleruby', 'truffleruby-head']
+        ruby: ['2.7', '3.0', '3.1', '3.2', '3.3', '3.4', 'debug', 'truffleruby', 'truffleruby-head']
```

```suggestion
        ruby: ['2.7', '3.0', '3.1', '3.2', '3.3', '3.4', 'ruby-head', 'debug', 'truffleruby', 'truffleruby-head']
```

Please don't remove ruby-head.